lerna.json

```diff
@@ -22,6 +22,17 @@
             "@azure/msal-angular"
         ],
         "stream": true
+    },
+    "bootstrap": {
+        "scope": [
+            "msal",
+            "@azure/msal-browser",
+            "@azure/msal-common",
+            "@azure/msal-node",
+            "@azure/msal-react",
+            "@azure/msal-angular"
```

Comment:
msal-node-extensions?

Reply:
once token-validation is published, does the user need to do extra steps to build?

Reply:
node-extensions isnt included in any of the other commands currently.

Reply:
> once token-validation is published, does the user need to do extra steps to build?

Once it is published, we can add it into this list.

Reply:
> node-extensions isnt included in any of the other commands currently.

The other commands are arguably not as important as the bootstrap. node-extensions should still consume the local copy of msal-common